lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -12,94 +12,55 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
+import { HEADER_RID } from "../../constants";
 import { BaseRequest, BaseResponse } from "../../framework";
-import { TypeNormalisedInput } from "./types";
+import { availableTokenTransferMethods } from "./constants";
+import { TokenTransferMethod, TokenType, TypeNormalisedInput } from "./types";
 
+const authorizationHeaderKey = "authorization";
```

Comment:
Shouldn't this be capital "A"?

Reply:
Header names are not case-sensitive according to the spec. 
Maybe we could add some processing internally to make sure, but for example, Axios always lowercases them.